docs/external plugins/Verifier/vulnerabilityreport.md

```diff
@@ -0,0 +1,333 @@
+# Vulnerability Report
+
+This document outlines how Ratify can be used to verify vulernability reports. The `vulnerabilityreport` verifier is added as a plugin to the Ratify verification framework. Currently the vulnerability report verifier supports the following report types:
+
+- Reports attached to the subject image as a referrer artifact
+- Reports in SARIF format
+- Reports generated by [Trivy](https://aquasecurity.github.io/trivy/v0.47/) or [Grype](https://github.com/anchore/grype)
+
+## Table of Contents
+
+* [Example Scenario](#example-scenario)
+* [Configuration](#configuration)
+* [Advanced](#advanced-scenario)
+
+## Example Scenario 
+
+Alice has a Kubernetes cluster. She installs Ratify with Gatekeeper on her cluster using the [Ratify quickstart guide](../../quick-start.mdx). She now wants to enable vulnerability report verification. In particular, she wants to make sure all container images used in K8s resources have at least one valid vulnerability report attached to the image. The report must:
+- be in SARIF format
+- generated from Trivy or Grype
+- created less than 24 hours ago
+- has no HIGH or CRITICAL severity level vulnerabilities in the report
+- does not contain the log4shell CVE 
+
+### Recording
+
+[![asciicast](https://asciinema.org/a/622368.svg)](https://asciinema.org/a/622368)
```

Comment:
There is a way to embed the player into the website page. Could you try it as follows?

``` 
<script async id="asciicast-622368" src="https://asciinema.org/a/622368.js"></script>
```

Reply:
Seems like this is nontrivial. I tried to change the file to mdx but I still cannot embed the script inside the file.